contracts/EventStore.sol

```diff
@@ -157,52 +134,4 @@ contract EventStore is Killable {
     return solidityEvents[_eventIndex].PropertyValues[_eventPropertyIndex].StringValue;
   }
 
```

Comment:
@OR13 I have questions about documenting the usage of the integrity hash, hopefully that has already been addressed.